pxi.py

```diff
@@ -170,107 +180,104 @@ def parse_xml(self, xml_str):
-                        #
-                        # self.return_data_str = str(return_data_queue
-
-                elif child.tag == "pause":
-                    # TODO: set state of server to 'pause';
-                    # i don't know if this a feature that currently gets used,
-                    # so might be able to omit this.
-                    pass
-
-                elif child.tag == "run":
-                    # TODO: set state of server to 'run';
-                    # i don't know if this a feature that currently gets used,
-                    # so might be able to omit this.
-                    pass
-
-                elif child.tag == "HSDIO":
-                    # set up the HSDIO
-                    try:
+
+                    elif child.tag == "HSDIO":
+                        # set up the HSDIO
                         self.hsdio.load_xml(child)
                         self.hsdio.init()
                         self.hsdio.update()
-                    except (HSDIOError, AssertionError, ValueError, KeyError) as e:
-                        self.handle_errors(e, "Initializing HSDIO")
+                        
+                    elif child.tag == "TTL":
+                        # self.ttl.load_xml(child)
+                        # self.ttl.init()
+                        pass
+                        
+                    elif child.tag == "DAQmxDO":
+                        # self.daqmxdo.load_xml(child)
+                        # self.daqmxdo.init()
                         pass
-                elif child.tag == "TTL":
-                    # self.ttl.load_xml(child)
-                    # self.ttl.init()
-                    pass
-                elif child.tag == "DAQmxDO":
-                    # self.daqmxdo.load_xml(child)
-                    # self.daqmxdo.init() # called setup in labview
-                    pass
-                elif child.tag == "timeout":
-                    try:
-                        # get timeout in [ms]
-                        self.measurement_timeout = 1000*float(child.text)
-                    except ValueError as e:
-                        self.logger.error(f"{e} \n {child.text} is not valid "+
-                                          f"text for node {child.tag}")
-
-                elif child.tag == "cycleContinuously":
-                    cycle = False
-                    if child.text.lower() == "True":
-                        cycle = True
-                    self.cycle_continuously = cycle
-
-                elif child.tag == "camera":
-                    pass
-                    # set up the Hamamatsu camera
-                    try:
+
+                    elif child.tag == "timeout":
+                        try:
+                            # get timeout in [ms]
+                            self.measurement_timeout = 1000*float(child.text)
+                        except ValueError as e:
+                            msg = f"{e} \n {child.text} is not valid "+
```

Comment:
Missing \ in line continuation. Preferably use a different method like parentheses